mypy/partially_defined.py

```diff
@@ -435,6 +439,10 @@ def visit_with_stmt(self, o: WithStmt) -> None:
             self.process_lvalue(idx)
         o.body.accept(self)
 
+    def visit_class_def(self, o: ClassDef) -> None:
+        self.process_definition(o.name)
```

Comment:
Why do we have `enter_scope()` for functions but not for classes? A variable defined in a class body will not be visible outside (as a variable, will still be available as an attribute of course)